### DISCUSSION TITLE
Request for GitHub Education rejects for unclear reason

### DISCUSSION THREAD

**Discussion by ADJURIC652** [community/community#142478] [GitHub Education]
### Hi everyone,

Hello,

I've sent about 4 or so requests trying to apply to GitHub Education. they have all been rejected but for reasons i dont understand.

The reasons are as followed:

"You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect, then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying."

I have given access to my location to github.com as well as completed personalizing my profile and completing the github billing information with my legal name as its stated on the affiliation document. im kind of lost on what is missing or what it is im doing wrong, can anyone advise?

**Comment by saadsohail05** [comment]
# Location Settings Check
1. Browser Location Access:
  - Chrome: Settings > Privacy & Security > Site Settings > Location
  - Firefox: Settings > Privacy & Security > Permissions > Location
  - Edge: Settings > Cookies and Site Permissions > Location

2. System Location Settings:
  - Windows: Settings > Privacy > Location
  - Mac: System Preferences > Security & Privacy > Location Services

# Profile Verification Steps
3. GitHub Profile Must Have:
  - Full legal name matching academic document exactly
  - School email added and verified
  - Profile picture
  - Complete bio
  - Location filled
  - Billing info matching legal name

4. Document Requirements:
  - Current academic affiliation document (not expired)
  - Name matches GitHub profile exactly
  - Shows current student status

5. Additional Steps:
  - Clear browser cache/cookies
  - Log out of GitHub
  - Disable any VPNs
  - Log back in
  - Try application again

Note: Each field must match EXACTLY - including middle names, capitalization, 
and spacing. Test location services on another site to verify they work.
